lib/asciidoctor/latex/converter.rb

```diff
@@ -89,35 +89,47 @@
 module Asciidoctor::LaTeX
   module Html5ConverterExtensions
 
-    def environment node
+    ENV_CSS = "+++<div style='line-height:1.5em;font-size:1.05em;font-style:oblique;margin-bottom:1.5em'>+++"
+    DIV_END = '+++</div>+++'
+    TABLE_ROW_END = '+++</tr></table>+++'
+
+    def info node
+
+      attrs =  node.attributes
+
+      warn "\n    node: #{node.node_name}".cyan
+      warn "   attrs: #{attrs}".cyan
+      warn "   title: #{attrs['title']}".cyan
+      warn "    role: #{attrs['role']}".cyan
+      warn "   level: #{attrs['level']}".cyan
+      warn " options: #{attrs['options']}".cyan
+      warn " type   : #{attrs['type']}".cyan
+      warn " caption: #{node.caption}".red
+      warn "      id: #{attrs['id']}".cyan
+      warn " content: #{node.content}".blue
```

Comment:
Is it really useful to have so many log messages? Logging should not replace debugging…

btw Ruby has multiline strings, you can write:

```ruby
warn "
  node: #{node.node_name}
  attrs: #{attrs}
  title: #{attrs['title']}
  role: #{attrs['role']}
  level: #{attrs['level']}
  options: #{attrs['options']}
  type : #{attrs['type']}
  caption: #{node.caption}
  id: #{attrs['id']}
  content: #{node.content}".cyan
```

or better using heredoc:

```ruby
warn <<-EOF
  node: #{node.node_name}
  attrs: #{attrs}
  title: #{attrs['title']}
  role: #{attrs['role']}
  level: #{attrs['level']}
  options: #{attrs['options']}
  type : #{attrs['type']}
  caption: #{node.caption}
  id: #{attrs['id']}
  content: #{node.content}
EOF.cyan
```

The indentation problem can be cleanly fixed using extension:

```ruby
class String
  def strip_heredoc
    indent = scan(/^[ \t]*(?=\S)/).min.try(:size) || 0
    gsub(/^[ \t]{#{indent}}/, '')
  end
end
```
from [active_support/core_ext/string/strip](https://github.com/rails/rails/blob/master/activesupport/lib/active_support/core_ext/string/strip.rb), that is used like:

```ruby
warn <<-EOF.strip_heredoc.cyan
  node: #{node.node_name}
  attrs: #{attrs}
  title: #{attrs['title']}
  role: #{attrs['role']}
  level: #{attrs['level']}
  options: #{attrs['options']}
  type : #{attrs['type']}
  caption: #{node.caption}
  id: #{attrs['id']}
  content: #{node.content}
EOF
```